### PR TITLE
small ui updates

### PR DESCRIPTION
**Pull request by mcraa**
Change-type: patch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +67,11 @@ rm -rf /var/lib/apt/lists/*
 apt-get update
 ```
 
-##### OpenSUSE LEAP & Tumbleweed install
+#### RPM based Package Repository (GNU/Linux x86/x64)
+
+> Detailed or alternative steps in the [instructions by Cloudsmith](https://cloudsmith.io/~balena/repos/etcher/setup/#formats-rpm)
```
Comment (lurch) [inline README.md:72]: Should this have been added to the "Redhat (RHEL) and Fedora-based Package Repository (GNU/Linux x86/x64)" section?

Comment (mcraa) [reply README.md:72]: zypper (opensuse) works with rpm too. From this line all `sh snippets` have the script with rpm.
At the same time, yes, it seems logical to have Redhat first in the rpm section.

**Comment by mcraa** [comment]
@balena-ci re-test

File: README.md [hunk 2]
```diff
@@ -124,6 +119,21 @@ rm /etc/yum.repos.d/balena-etcher.repo
 rm /etc/yum.repos.d/balena-etcher-source.repo
 ```
 
+#### OpenSUSE LEAP & Tumbleweed install (zypper)
+
+```sh
+curl -1sLf \
+  'https://dl.cloudsmith.io/public/balena/etcher/setup.rpm.sh' \
+  | sudo -E bash
```
Comment (lurch) [inline README.md:127]: Reading through the install-instructions for the other OSes, it seems that this `curl ... | bash` line just adds the CloudSmith repositories to the system repositories, and then there's some other package-commands (`apt-get` / `dnf` / `yum`) to actually install the `balena-etcher-electron` package? So perhaps OpenSUSE is similar, and that also needs an additional package-command (`zypper`?) to actually install balenaEtcher?  (I can't be sure, as I've never used OpenSUSE myself)

Comment (mcraa) [reply README.md]: thanks, quite crowded part, i did not even notice it is missing (and outdated)

**Comment by mcraa** [comment]
@balena-ci re-test!

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by lurch** [comment]
"If at first you don't succeed, try, try again" ?  :laughing:

**Comment by mcraa** [comment]
@lurch i wanted to comment by myself "best PR chat ever"

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci re-test

**Comment by mcraa** [comment]
@balena-ci I self-certify!

**Comment by lurch** [comment]
best PR chat ever!

**Pull request merged**